bundles/bcache-tools

```diff
@@ -1,8 +1,8 @@
 # [TITLE]: bcache-tools
-# [DESCRIPTION]: bcache is a way to pair a fast SSD with a slow HDD to create a hybrid disk
+# [DESCRIPTION]: bcache is a way to pair a fast SSD with a slow HDD to create a hybrid disk.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Tools and Utilities
```

Comment:
TAGS are keywords, not sentences, so the word "AND" is not appropriate

it's also the categories that these show up in the UI so lets make sure that these are the ones we want... like how are "Tools" and "Utilities" different things/tabs ?

Reply:
It was my understanding that the categories in the UI would match, so the category is "Tools and Utilities" in this case. I may have missed something though.

Reply:
"Tools and Utilities" is a category per Endre V. 
Chris N., @pixelgeek, may also know.  Not sure why "and" was chosen.